ui_framework/components/icon/kui_icon.js

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+
+export function KuiIcon({ className }) {
+  const classNames = ['kuiButton__icon', 'kuiIcon', className];
+  return <span aria-hidden="true" className={ classNames.join(' ') }/>;
```

Comment:
it's yet another dependency just for a single function...  we can copy the function, put it in our utils and maintain it ourselves.